pyproject.toml

```diff
@@ -103,6 +103,8 @@ dev = [
     "nox",
     # For coverage
     "pytest-cov",
+    # For doctest
+    "pytest-doctestplus",
```

Comment:
Could you add an entry for this to the table in the installation guide? Here: https://docs.pybamm.org/en/latest/source/user_guide/installation/index.html#dev-dependencies

Reply:
I've also modified other outdated entries as well.